site/content/flowcharts/lonecoder.md

```diff
@@ -24,5 +24,62 @@ graph TD;
   class Link,B,C,F linkedBox
 {{< /mermaid >}}
 
-To read about this workflow in detail, please see
-[single developer process](/dev-review/recipes/lonecoder)
+### Find a reviewer
+
+There is no code review without (a) reviewer(s). The good news is,
+almost any researcher who code is a good candidate for being
+one. 
+
+Potential reviewers are not far: in your own research group, others
+you collaborate with or your local Research Software Engineering
+group. They can also be found in communities outside your institution
+or specific research domain. In the [Find a reviewer](https://researchcodereviewcommunity.github.io/dev-review/recipes/find_a_reviewer/) phase, we look at
+different pools of potential reviewers, what makes a suitable
+reviewer, and ways of getting in touch.
+
+### Meet and agree on objectives
+
+A code review can improve code in a lot of different ways, but trying
+to do everything at once is rarely effective. In the [Meet and agree
+on
+objectives](https://researchcodereviewcommunity.github.io/dev-review/recipes/meet_and_agree_on_objectives/)
+phase, participants make it clear to others people involved why their
+are pariticpating.
+
+The goal of this second phase is for all the participants to agree on
+the purpose of the review. This is critical to a constructive and
+useful discussion during the actual review process. Key questions are:
+- What is the author expecting from the review?
+- What should the review enable? Better readability? Knowledge
+  transfer? Better performance?
+- What part of the code should we look at?
+
+
+### Meet to review code
```

Comment:
I think this is the only part where what you have done and what I saw for the high level flowchart differ. I broke this section up into sub parts on the flow chart (which do align with the sub tasks you have below). This is what it would look like:

![Screenshot from 2021-11-10 10-28-49](https://user-images.githubusercontent.com/6177028/141097010-52bfb2b8-c460-4218-8428-d85cfc0a7123.png)

I am interested to get everyone's opinion on this version (it is what I wanted to show at our last meeting). My feeling is that it probably has a bit too much detail for the highest level. @tlestang @NickleDave @fcooper8472 @bdemchak @mxochicale

Reply:
I personally like that @bielsnohr . Writing the "Meet to review code" outline felt awkward because this is actually the core of the business, and ti felt more details would have been good to add. 

We briefly discussed this today, could you open a new issue with that diagram and a description?